ui/js/component/transactionList/view.jsx

```diff
@@ -23,6 +24,16 @@ class TransactionList extends React.PureComponent {
     return !filter || filter == transaction.type;
   }
 
+  isRevokeable(txid, nout) {
+    // a claim/support/update is revokable if it
+    // is in my claim list(claim_list_mine)
+    return this.props.myClaims.has(`${txid}:${nout}`);
```

Comment:
This is fine, but if we find ourselves doing this repeatedly (which I think we are), it may make sense to have outpoint be a first-class property so it's not rebuilt in 16 million different places.

Reply:
Not doing this, as per the discussion in the meeting.